### ISSUE TITLE
Primitive type helpers

### ISSUE THREAD

**Issue by brendt** [tempestphp/tempest-framework#288]
- Arr
- Str
- Path
- Int

**Comment by Tresor-Kasenda** [comment]
Speaking of primitives, can I say that you want to create classes for each primitive and then add helper? If yes i can contribute to this issue

**Comment by brendt** [comment]
ArrayHelper PR is here: https://github.com/tempestphp/tempest-framework/pull/434

**Comment by aidan-casey** [comment]
@brendt - Does it make sense to move the path helper to a `Filesystem` component? I am re-looking at the filesystem component, so just thinking about that now.

**Comment by brendt** [comment]
I don't think so. I want this path helper to be just that: a small wrapper around pathinfo. I might also want to add something like `path('…')->put($input)` (which ensures the directory exists), but nothing more than that. 

I think a proper filesystem with multiple drivers is still needed, but it's not what this helper is for.

**Comment by Tresor-Kasenda** [comment]
it's a first for me to contribute on an open source project i'd like to have details on the path helper primitive in the case of path

**Comment by brendt** [comment]
Hi @Tresor-Kasenda thanks for asking, I noticed you asked on stream yesterday and I forgot to answer, my bad! 🫣

Now that Aidan has been working on the filesystem, I might actually reconsider my stance on the matter, and maybe `path` should be a simple wrapper that returns a filesystem instance. We'll have to wait until Aidan finished #441 before we can continue with this one though
